client/src/components/ImportData/CSVUpload.jsx

```diff
@@ -1,41 +1,72 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useContext } from 'react';
 import axios from 'axios';
+import {UserContext} from '../../App.jsx';
+
+
+const CSVUpload = ({handleData, handleDatasetId}) => {
+  let user = useContext(UserContext);
 
-const CSVUpload = () => {
   const [title, setTitle] = useState('');
-  const [ownerId, setOwnerId] = useState('4');
-  const [teamId, setTeamId] = useState('3');
+  const [teamId, setTeamId] = useState(1);
+  const [teams, setTeams] = useState([]);
+
+  useEffect(() => {
+    axios.get('/teams')
+    .then((teams) => {
+      setTeams(teams.data);
+    })
+  }, []);
 
   const handleChange = (e) => {
     setTitle(e.target.value);
   }
 
+  const handleTeamChange = (e) => {
+    setTeamId(e.target.value);
+  }
+
   const handleSubmit = (e) => {
     e.preventDefault();
     var file = document.getElementById('file');
     const formData = new FormData();
     formData.append('csv', file.files[0]);
     formData.append('title', title);
-    formData.append('ownerId', ownerId);
+    formData.append('ownerId', user.user.id);
     formData.append('teamId', teamId);
     console.log(formData);
     const res = axios.post(`/datasets/`, formData, {
       headers: {
         'Content-Type': 'multipart/form-data'
       }
     })
-    .then(res => console.log(res));
+    .then(res => {
+      console.log(res.data.id);
+      handleDatasetId(res.data.id);
+      return axios.get(`/datasets/${res.data.id}`)
+    })
+    .then(dataset => {
+      handleData(dataset.data.datapoints);
+    })
   }
 
   return (
     <form>
       <input type='file' id='file'></input>
       <input type='text' value={ title } onChange={ handleChange }></input>
+      <select onChange={handleTeamChange} value={teamId}>
```

Comment:
Since these event handlers are just invoking setState, you could do `onChange={(e} => setState(e.target.value)}` if you wanted to avoid creating those functions up above